Library/Formula/highlighting-kate.rb

```diff
@@ -0,0 +1,35 @@
+require "language/haskell"
+
+class HighlightingKate < Formula
+  include Language::Haskell::Cabal
+
+  homepage "https://github.com/jgm/highlighting-kate"
+  url "https://github.com/jgm/highlighting-kate/archive/0.5.14.tar.gz"
+  sha256 "0c9ad39ee1b123b655434ff3a189c7aae634057711e120c08c01e4c310e959fd"
+
+  head "https://github.com/jgm/highlighting-kate.git"
+
+  depends_on "ghc" => :build
+  depends_on "cabal-install" => :build
+  depends_on "gmp"
```

Comment:
It isn't:
```
ld: warning: directory not found for option '-L/cores/homebrew/Temp/ghc20150422-76234-gpihj1/ghc-7.10.1/gmp-static'
ld: library not found for -lgmp
clang: error: linker command failed with exit code 1 (use -v to see invocation)
linking dist/dist-sandbox-d7a63a4d/build/Text/Regex/PCRE/Wrap_hsc_make.o failed (exit code 1)
```
...because its still looking for `libgmp.a` in the ghc buildpath. #39025